internal/pipeline/parser_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+func ptr[T any](x T) *T { return &x }
+
 func TestParserParsesYAML(t *testing.T) {
 	envMap := env.FromSlice([]string{`ENV_VAR_FRIEND="friend"`})
```

Comment:
```suggestion
	envMap := env.FromSlice([]string{`ENV_VAR_FRIEND=friend`})
```
I think you should change the test data to be like this. `os.Environ()` won't put quotes if there was an environment variable like this, [even if the value contains spaces](https://go.dev/play/p/p8seo0PUTk6), and this map is meant to mimic its output. You'll have to change the expected results too.

Reply:
Done - and where that removes `"`, I've updated <code>\`</code> strings to `"`.